src/main/java/gov/cms/madie/models/cqm/datacriteria/AdverseEvent.java

```diff
@@ -6,19 +6,33 @@
 import gov.cms.madie.models.cqm.datacriteria.basetypes.Code;
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 import lombok.experimental.SuperBuilder;
 
-import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
 import java.util.List;
 
+import org.springframework.format.annotation.DateTimeFormat;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
 @Data
 @SuperBuilder
 @NoArgsConstructor
 @AllArgsConstructor
+@Getter
+@Setter
 public class AdverseEvent extends DataElement {
-  private LocalDateTime authorDatetime;
-  private LocalDateTime relevantDatetime;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+  private ZonedDateTime authorDatetime;
+
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
```

Comment:
does this not work?

```suggestion
  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = LocalDateTimeFormatConstant.LOCAL_DATE_TIME_PATTERN)
```

Reply:
@adongare Not sure what happened, I used the constants for all of those, now the code goes back to hard coded.

Reply:
@adongare I changed all of those back to constants.